### DISCUSSION TITLE
UI: Fully fit pictures to screen?

### DISCUSSION THREAD

**Discussion by Alestrix** [photoprism/photoprism#2105] [Q&A] [answered]
Hi,

is there a function to display the pictures in full screen? I mean really screen-fitting full screen? Because when using the full screen button/feature there is still a significant border around the picture. 
One can click on the zoom icon, but this has two issues:
- On lower resolution screens this will zoom too far into the picture
- It doesn't play well with slide shows

Are there any features or tricks I missed?

Thanks!
Alex

**Comment by lastzero** [comment, marked as answer]
We need to develop a new, custom lightbox. There's an existing issue for this.

**Reply by Alestrix** [reply]
Perfect!
With the search term "lightbox" I was able to find the corresponding issue #1307, and specifically #1379.
